src/Bridge/Doctrine/Persister/ObjectManagerPersister.php

```diff
@@ -87,7 +87,7 @@ public function persist($object)
                 throw $exception;
             }
 
-            if (null !== $generator && false === $generator->isPostInsertGenerator()) {
```

Comment:
Found this bug when using a "cascade persist" relationship :

An item (lets say it is a `Category`) persisted in database with an id, the code lines 67 to 72 will remove the generator from the metadata.
As it is a `Doctrine\ORM\Id\IdentityGenerator` (isPostInsertGenerator == true), the generator will not be reset.
After that, you persist an `Article` with a new `Category`, the Category metadata will still be of type `GENERATOR_TYPE_NONE`, and Doctrine will throw an exception because id generation strategy is wrongly set to None.

I just commented the `isPostInsertGenerator` to open discussion because I do not really see the point of this test.
For me, if the generator has been changed, it is needed to reset it to its original value, don't  you think ?